docs/topic/platform.notifications.ingress.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+---
+\[[Front page](../overall-architecture.html)\] \[[Overall architecture](../overall-architecture.html)\]
+
+
+
+# Topic: `platform.notifications.ingress`
+
+
+
+## Topic description
+
+When changes (new issues) has been found in *CCX Notification Service*, notification message is sent into this Kafka topic for given (actually processed) cluster. Notification backend is configured as consumer. More info about Notification backend can be seen on this page [https://redhatinsights.github.io/ccx-notification-service/event_targets.html](https://redhatinsights.github.io/ccx-notification-service/event_targets.html).
+
+
+
+## Messages format
+
+* Format is described on page [https://core-platform-apps.pages.redhat.com/notifications-docs/dev/user-guide/send-notification.html#_kafka](https://core-platform-apps.pages.redhat.com/notifications-docs/dev/user-guide/send-notification.html#_kafka)
+
+
+
+## Schema version
+
+1 (unofficial)
+
+
+
+## Schema description
+
+N/A at this moment
```

Comment:
Schema is also described in https://core-platform-apps.pages.redhat.com/notifications-docs/dev/user-guide/send-notification.html#_kafka , right?

Reply:
yup, actually yes :)

Reply:
fixed